modules/nw-networkpolicy-audit-configure.adoc

```diff
@@ -0,0 +1,228 @@
+[id="nw-networkpolicy-audit-configure_{context}"]
+= Configuring network policy auditing
+
+As a cluster administrator, you can customize network policy audit logging to suit your environment.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+* To customize the network policy audit logging configuration, enter the following command:
++
+[source,terminal]
+----
+$ oc edit network.operator.openshift.io/cluster
+----
++
+[TIP]
```

Comment:
I realize that we just went through this exercise to add these TIPs, but the purpose was simply to make sure we dispelled any mysteries for customers who are new to GitOps.  In this case, I believe you could add the YAML as part of the `oc edit` command.  As always, up to you.

Reply:
@mikemckiernan, true, how this manifests is... interesting, as a documentation-wide policy. I don't _think_ it does any harm though, other than bloating these procedures.